client/my-sites/themes/themes-banner/style.scss

```diff
@@ -68,6 +68,30 @@
 		margin-top: 1em;
 	}
 
+	.button.themes-banner__close {
+		display: block;
+		position: absolute;
+		background: rgba( white, 0.66 );
+		border-radius: 0;
+		border: none;
+		padding: 1px 0;
+		width: 1.7em;
+		line-height: 0.9;
+		z-index: 4;
+		top: 0;
+		right: 0;
+
+		&:hover {
+			cursor: pointer;
+		}
+
+		svg.gridicons-cross-small {
```

Comment:
No need for the `svg` tag selector -- just go with the class name :slightly_smiling_face: